Content.Shared/Storage/Components/SharedEntityStorageComponent.cs

```diff
@@ -146,9 +146,19 @@ public EntityStorageComponentState(bool open, int capacity, bool isCollidableWhe
     }
 }
 
+// Why does this even pass the entity being inserted if it's raised to the entity being inserted...
+/// <summary>
+/// Raised to the entity being inserted whenever checking if an entity can be inserted into a container.
```

Comment:
```suggestion
/// Raised on the entity being inserted whenever checking if an entity can be inserted into an entity storage.
```
an entity storage is a container, but not all containers are entity storages